compose/ui/ui/build.gradle

```diff
@@ -175,12 +175,10 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
                 api project(":compose:ui:ui-unit")
                 api project(":compose:ui:ui-util")
                 implementation(project(":compose:ui:ui-backhandler"))
-
-                // TODO(igor): figure out how to avoid downgrading runtime to 1.6 in case of project dependencies in mpp demo + pinned lifecycle here
-                implementation(project(":lifecycle:lifecycle-common"))
-                implementation(project(":lifecycle:lifecycle-runtime"))
-                implementation(project(":lifecycle:lifecycle-runtime-compose"))
-                implementation(project(":lifecycle:lifecycle-viewmodel"))
+                api("org.jetbrains.androidx.lifecycle:lifecycle-common:2.8.4")
```

Comment:
Why it is `api`?

Reply:
Part of public API - `LocalLifecycleOwner`